src/nvim/log.h

```diff
@@ -4,6 +4,8 @@
 #include <stdio.h>
 #include <stdbool.h>
 
+#include "nvim/os/os_defs.h"
```

Comment:
AppVeyor worker images are updated to VS2015 Update 1 (dec-4): http://www.appveyor.com/updates. By default we will get Update 1 in `PATH`. To revert back to RTM, use `os: Previous Visual Studio 2015` (but we probably don't want to go back :smile:)